### ISSUE TITLE
Various editorial issues and suggestions for current crawling draft

### ISSUE THREAD

**Issue by aisaac** [IIIF/discovery#14]
* Scoping remarks already discussed in call of 2018-02-07, but I'm repeating in case reference to specific parts of text would help*

1. The title is very generic: ("IIIF Resource Discovery")
2. The intro mentions "indexing" at the same level as "crawling"
3. The Approach mentions "the IIID Discovery charter and use cases" in their entirety.

*new remarks and suggestions*

4. Approach section mentions "notification of changes" in the bullet list about ResourceSync, but there's no real notification in the current spec, so the bullet could go away.

5. "follow the best practices defined in the W3C" reads quite vague. And while AS do follow many best practices, they don't follow all of them (for example, re-using existing W3C specs like PROV instead of creating an entire vocabulary from scratch)

6. We could add "This specification covers the listing of core IIIF resources, namely IIIF manifests and IIIF collections" as the first sentence of "List of Resources" section to make it clear which "IIIF resources we're focusing on. By the way I'm adding "IIIF" in front of them so as to distinguish with AS2 Collection which would come just afterwards.

7. Names of instances (in the RDF sense) sound a bit generic, like `https://example.org/iiif/discovery.json` (which hints to cover every discovery need - it may eventually, but not right now!) or `http://example.org/museum` (which sounds like a type/class name). Could they be a bit more specific, e.g. `https://example.org/iiif/updateCollection.json` or `http://example.org/myMuseum`

8. “to see if it has changed” could be removed - or downplayed - in the “Usage” of Level 0: in the most basic scenario, one would probably harvest all the manifests without checking if they’ve changed. 

9. In the intro for Level 1 “the first pages, once finished, become static resources” could be reworded as “the first pages, once finished, can become static resources”. The current wording hints that it would always be the case. It shouldn’t hold in the scenario where a publisher decides to have a manifest appear only once with its most recent update, should it?

10. Is there a specific reason why usage of Level 1b is worded differently from the usage of level 1? I don’t see much difference in the message. If it’s the same message, it should be the same sentence (though it’s not very exciting...)

11. Can there be a way to highlight in the examples what’s being added at every level? As they build on top of each other, it would be a powerful aid to understand the whole story. NB: obviously it is a nice-to-have, and may have to be dropped in case it would be considered bad for accessibility.

12. The beginning of the “Processing Algorithm” section is heavily focused on indexing. But this is only of the scenarios (although a key one for us). There might be harvesters out there that just want to display images, perhaps do the filtering themselves, and not necessarily “build an index of the resources to allow them to be discovered” in the way an aggregator like Europeana would do.

13. The Target Resource Algorithm is probably tuned too much for indexing (a set of SHOULDs within a SHOULD) if we intend to make the scope of the document less centered on this goal.

**Comment by aisaac** [comment]
@azaroth42 I've reviewed these comments, which were about
https://github.com/IIIF/discovery/blob/master/source/api/harvest/0.1/index.md
against the latest draft at
http://preview.iiif.io/api/discovery/api/discovery/0.1/

Here is the outcome:
The scoping comments 1-4 still apply to some extent: The intro has the paragraph “The first step towards enabling the discovery of IIIF resources” but I think it’s not formal enough and there’s “coming soon” for section 4 notification. I’d be in favour of removing section 4 and drop a front note like the  ‘status warning’ (maybe add to that existing note?) saying that we address only 1A in our charter in this document, and making it clear that it will be complemented by other specs, which could be in the same API or in other doc (just like our answer to 4 in the charter will probably be). I know I’m a pain, but I think it will be less painful than having to handle comments/misunderstandings on this, later.

5, 6 do not apply anymore

7 applies only for https://example.org/activity/collection and https://example.org/person/admin, which look like generic names/classes, not like instances.

8, 9, 10 do not apply anymore

11 still applies, but as the examples for the levels seem shorter this is much less important. 

12 still applies, albeit perhaps differently. The paragraph on indexing is kind of the only introduction to the section (and thus the algorithms) while the algorithms’ goals are much more generic (just building lists of resources to be further process, e.g. displayed, indexed, etc). I would suggest to put the paragraph elsewhere, or park it for when the group wants to address indexing in a more systematic way (I’m not saying that the technical answer would be much different from what you’ve written, but it may be written differently, e.g., in a different section).

13 does not apply anymore

**Comment by azaroth42** [comment]
Thanks Antoine! Agreed with all, and changes made for all but 11.  We don't have any way to highlight content inside the &lt;pre> blocks for example at the moment.  Let me know if you think the issues are addressed:  http://preview.iiif.io/api/discovery/api/discovery/0.1/

**Comment by aisaac** [comment]
Thanks Rob!
For 12 the paragraph still seems in the same position in where it biases the reading of the otherwise agnostic algorithm (I mean agnostic to the type of processing done once the resources are found)

**Comment by azaroth42** [comment]
True. Shuffled it down to its own section.
